src/libraries/System.Data.Common/src/System/Data/Common/DbConnectionStringBuilder.cs

```diff
@@ -556,6 +564,7 @@ private PropertyDescriptorCollection GetProperties(Attribute[] attributes)
             return new PropertyDescriptorCollection(filteredPropertiesArray);
         }
 
+#nullable disable
```

Comment:
Why is this disabled?

Reply:
Oh, when I originally did this PR System.ComponentModel wasn't annotated yet. Will do this.

Reply:
Actually ICustomTypeDescriptor is part of System.ComponentModel.TypeConverter, which indeed isn't annotated yet. Am going to leave this for now and will revisit after that gets annotated.